packages/shared/lib/core/wallet/utils/outputs/preprocessGroupedOutputs.ts

```diff
@@ -49,15 +47,22 @@ function isTransactionIncoming(
     outputs: OutputData[],
```

Comment:
This is named after a transaction but uses outputs, also it should be called something to do with direction not isIncoming